cmd/create/accountroles/cmd.go

```diff
@@ -383,7 +412,7 @@ func run(cmd *cobra.Command, argv []string) {
 	}
 
 	input := buildRolesCreationInput(prefix, permissionsBoundary, r.Creator.AccountID, env, policies,
-		policyVersion, path)
+		policyVersion, path, isHcpSharedVpc)
```

Comment:
Would rather not include a bool param if possible

Reply:
If there is too much refactoring required we can move ahead

Reply:
I think it would be best to move ahead-
Im not sure I see a benefit from making one function which returns a struct containing `hostedCp: false` and another func returning the same struct with `hostedCp: true` - would like to discuss further for the future at another point 🙂